src/hooks/map/useMap.ts

```diff
@@ -39,16 +50,33 @@ function useMap(): MapHookType {
       }
       return;
     }
-    initHistoryStatus();
+    dispatch(initHistory());
+    dispatch(initMarker());
     setFlag(true);
   };
 
+  const printMarker = (): void => {
+    if (!marker) return;
+
+    marker.markers.forEach((item) => {
+      registerMarker({
+        id: item.id,
+        text: item.text,
+        lngLat: { lng: item.lng, lat: item.lat },
+      });
+    });
+  };
```

Comment:
아래처럼 이 함수에 `marker`만 인자로 넘겨서 `useMarkerFeature`에서 가져오는 방식이라면
좀더 useMap 자체 로직에 집중될 것 같은데 
여기서 선언해야하는 이유가 있을까요?

```suggestion
	const { printMarker, registerMarker } = useMarkerFeature();
	...
	printMarker(marker);
```

Reply:
음 printMarker 함수 내부 로직을 보시면 registerMaker를 연속적으로 부르는 것으로 로직의 특이성과 같은 이유로 함수를 나누었다기보다, 초기화 과정에서 화면에 그려주는 과정을 좀 더 의미를 파악하기 쉽게 하기 위해 만든 것입니다. 따라서 useMarkerFeature로 따로 분리하기 보다 불리는 곳에서 선언해주었습니다..!